l10n_es_aeat_sii_oca/models/sii_mixin.py

```diff
@@ -151,6 +152,16 @@ def _compute_sii_registration_key_code(self):
     def _compute_sii_enabled(self):
         raise NotImplementedError
 
+    @api.model
+    def _is_unsupported_search_operator(self, operator):
+        return operator not in ("=", "!=")
```

Comment:
Creo que tienes que volver a lo que había antes, porque no estás probando `!= True`, si no que estás probando `= False`, que no es lo mismo.

```suggestion
        return operator != "="
```

Reply:
Este cambio es el que sugeriste en https://github.com/OCA/l10n-spain/pull/3836#discussion_r1879951200 Se deben tener en cuenta ambos operadores porque cuando se filtra por ese campo se utiliza realmente: `('sii_enabled', '!=', True)` tal y como indicaba en https://github.com/OCA/l10n-spain/pull/3836#discussion_r1879873183
![ejemplo](https://github.com/user-attachments/assets/6c9e42f7-b53e-4ef9-88d2-402e8c06269d)

Reply:
Pero insisto en que no va a funcionar tal cual porque los AND pasarían a ser OR, y tú en ningún test lo estás probando que funcione.